docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+.. post:: October 9, 2017
+   :tags: newsletter
```

Comment:
I'd love to see tags based on the content of the newsletter, so we can make things easier to find in the future. Like: https://github.com/writethedocs/www/blob/master/docs/blog/newsletter-february-2017.rst

Reply:
Ooh, yeah. I can dig it.